### PR TITLE
Fix broken link referencing David Joyners page in FAQ

### LINKED ISSUE #140: Links in FAQ to David Joyner's website are broken
Under section 7 of the GAP FAQs we refer to two pages, one by David Joyner and one by Alexander Hulpke. Both links don't work at the moment.

UPDATE: Hulpke's links are now fixed.

### PR DESCRIPTION
**Pull request by DominikBernhardt**
Fixes #140 .

As by the comment of @hulpke in #141 , I deleted his page from the FAQs.

### CODE REVIEW

**Comment by alex-konovalov** [comment]
Maybe @hulpke's material can be rescued from https://web.archive.org/web/20181111181354/http://www.math.colostate.edu/~hulpke/GAPQA/ as @fingolfin discovered?

**Comment by hulpke** [comment]
Thank you @alex-konovalov, I have restored the files.

File: Doc/Learning/learning.mixer [hunk 1]
```diff
@@ -87,21 +87,11 @@ Of this the section 'Graph Theory using the GRAPE package' was contributed by
   <li> 
     <mixer person="David Joyner" data="name_link"/>
     is collecting a list of frequently asked questions about 
-    <a href="https://yetanothermathblog.com/2017/08/09/how-do-i-construct-in-gap/">
+    <a href="https://yetanothermathblog.com/the-toric-package-for-gap/how-do-i-construct-in-gap/">
     Constructions</a> of various mathematical objects in GAP
     with fully worked out GAP code answering them. This 
     collection is specially recommended for newcomers to the system . 
   <br />&nbsp;  
-  </li>
```
Comment (alex-konovalov) [inline Doc/Learning/learning.mixer]: This can now be restored - thanks @hulpke for recovering the files.

File: Doc/Learning/learning.mixer [hunk 2]
```diff
@@ -87,21 +87,11 @@ Of this the section 'Graph Theory using the GRAPE package' was contributed by
...
-  <li>
```
Comment (alex-konovalov) [inline Doc/Learning/learning.mixer]: @DominikBernhardt please restore this block - thanks to @hulpke, the material  has been recovered

File: Faq/faq.mixer [hunk 3]
```diff
@@ -142,20 +142,12 @@
     <a href="http://www.usna.edu/Users/math/wdj/">
     David&nbsp;Joyner</a> is collecting a list of frequently asked questions
     about
-    <a href="http://www.wdjoyner.org/blog/2015/05/06/basic-gap-constructions/">
+    <a href="https://yetanothermathblog.com/the-toric-package-for-gap/how-do-i-construct-in-gap/">
     Constructions</a> of various mathematical objects in GAP
     with fully worked out GAP code answering them. This
     collection is specially recommended for newcomers to the system .
     </li>
-    <li>
```
Comment (alex-konovalov) [inline Faq/faq.mixer]: @DominikBernhardt same here

**Comment by alex-konovalov** [comment]
@DominikBernhardt also, Travis test failed - please fix the syntax
```
Processing:/home/travis/build/gap-system/GapWWW/Faq/faq.mixer
Traceback (most recent call last):
  File "./Mixer/mixer.py", line 400, in <module>
    os.path.walk(MIXERROOT,walker,None)
  File "/usr/lib/python2.7/posixpath.py", line 239, in walk
    walk(name, func, arg)
  File "/usr/lib/python2.7/posixpath.py", line 231, in walk
    func(arg, top, names)
  File "./Mixer/mixer.py", line 398, in walker
    workerfunc(dirname[l:],f[:-6])
  File "./Mixer/mixer.py", line 315, in workerfunc
    main = maxml.parse_file_for_rewrite(source)
  File "/home/travis/build/gap-system/GapWWW/Mixer/maxml.py", line 84, in parse_file_for_rewrite
    return objectify(p(s))
pyRXP.error: Error: Mismatched end tag: expected </ul>, got </li>
 in unnamed entity at line 200 char 7 of [unknown]
Mismatched end tag: expected </ul>, got </li>
Parse Failed!
```

File: Faq/faq.mixer [hunk 4]
```diff
@@ -142,20 +142,12 @@
...
-    <mixer person="Alexander Hulpke" data="name_link"/>
-    has collected user questions (mostly from the GAP
-    Forum) about mathematical applications of
-    GAP together with the corresponding answers. See
-    <a href="http://www.math.colostate.edu/~hulpke/GAPQA/">
-    Some&nbsp;GAP&nbsp;Questions</a> on his home page.
-    </li>
-  </ul>
+  /ul>
```
Comment (fingolfin) [inline Faq/faq.mixer]:
```suggestion
  </ul>
```

**Comment by alex-konovalov** [comment]
@DominikBernhardt ping?

Also, @wdjoyner suggests to delete "5.7 Can I convert a file obtained from the GAP LogTo command into a Latex file?"

**Comment by DominikBernhardt** [comment]
Imho this is a useless PR now as the old links have been restored. I'm going to close this.

**Comment by wdjoyner** [comment]
On Mon, Mar 23, 2020 at 7:41 AM Dominik Bernhardt <notifications@github.com>
wrote:

> Imho this is a useless PR now as the old links have been restored. I'm
> going to close this.
>

FYI, 5.7 still has a bad link at
https://www.gap-system.org/Faq/faq.html#5.7

**Comment by alex-konovalov** [comment]
@DominikBernhardt I think that mixes two things - @wdjoyner's links which should be updated/removed, and @hulpke's which are restored.

**Pull request closed without merging**